huaweicloud/services/apigateway/resource_huaweicloud_api_gateway_environment.go

```diff
@@ -0,0 +1,147 @@
+package apigateway
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/apigw/shared/v1/environments"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceEnvironment() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceEnvironmentCreate,
+		ReadContext:   resourceEnvironmentRead,
+		UpdateContext: resourceEnvironmentUpdate,
+		DeleteContext: resourceEnvironmentDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"create_time": {
```

Comment:
please define it as **created_at**

Reply:
I use "create_time" due to it is written in the API document. Do we have to ues "create_at"?

Reply:
> I use "create_time" due to it is written in the API document. Do we have to ues "create_at"?

I use "create_time" due to it is written in the API document. Do we have to ues "create_at"?

Reply:
> please define it as **created_at**

I use "create_time" due to it is written in the API document. Do we have to ues "create_at"?

Reply:
> please define it as **created_at**

I have fixed this problem, thanks for your valuable suggestions